runtime/testnet/src/config/contracts.rs

```diff
@@ -36,7 +25,7 @@ impl<T: pallet_contracts::Config> Randomness<T::Hash, BlockNumberFor<T>> for Dum
 parameter_types! {
 	pub const DepositPerItem: Balance = deposit(1, 0);
 	pub const DepositPerByte: Balance = deposit(0, 1);
-	pub Schedule: pallet_contracts::Schedule<Runtime> = schedule::<Runtime>();
+	pub Schedule: pallet_contracts::Schedule<Runtime> = Default::default();
```

Comment:
isn't it possible that this could break existing contracts already deployed on testnet? If the limits are now lower, contracts used for development could break.

Reply:
You are right, I asked for feedback to maintainers of the substrate contracts node and in the meantime changed it back.